corehq/apps/app_manager/views/forms.py

```diff
@@ -721,14 +723,24 @@ def get_form_view_context_and_template(request, domain, form, langs, current_lan
         valid_index_names.append(USERCASE_PREFIX[0:-1])     # strip trailing slash
 
     form_has_schedule = isinstance(form, AdvancedForm) and module.has_schedule
+
+    try:
+        case_properties_map = get_all_case_properties(app)
+        usercase_properties_map = get_usercase_properties(app)
+    except CaseError as e:
+        case_properties_map = {}
+        usercase_properties_map = {}
+        if not has_case_error:
```

Comment:
Don't you mean `if has_case_error`?

Reply:
Oh! Sorry I misread the context.

Reply:
no - if `has_case_error` is true, this message will already have been a message added

Reply:
jinx :)